business/checkers/virtualservices/subset_presence_checker.go

```diff
@@ -11,10 +11,10 @@ import (
 )
 
 type SubsetPresenceChecker struct {
-	Conf             *config.Config
-	DestinationRules []*networking_v1.DestinationRule
-	Namespaces       []string
-	VirtualService   *networking_v1.VirtualService
+	Conf           *config.Config
+	DRSubsets      map[string]map[string]kubernetes.Host
```

Comment:
For example, this should be a type name that I can easily navigate to in the IDE and know what's going on.

Reply:
good point, let me add that